scripts/frr/configs/commands/isis.json

```diff
@@ -12,6 +12,13 @@
     "/interfaces/@element/isis/password/md5": "isis password md5 {/@text}",
     "/interfaces/@element/isis/network": "isis network {/@text}",
     "/interfaces/@element/isis/passive": "isis passive",
+    "/interfaces/@element/isis/topology/ipv4-mgmt": "isis topology ipv4-mgnt",
+    "/interfaces/@element/isis/topology/ipv4-multicast": "isis topology ipv4-multicas",
```

Comment:
It's functionally fine, but it would be better to list the vtysh command. `ipv4-multicas` -> `ipv4-multicast`.

Reply:
Resolved.